### ISSUE TITLE
Disabled In-Line Titles Prevent Titles from Appearing in Gallery View

### ISSUE THREAD

**Issue by trevware** [pashashocky/obsidian-note-gallery#10]
Hey there!

Disabling titles under `Appearance` will prevent titles from appearing in the Gallery View, despite the flag being set to `True` within the config. Not sure if intended, but just wanted to call it out. It would be nice (or rather, preferable) that that option overrides the Appearance toggle, since it's nice to see the titles in the Gallery view, but not so much in the Editor itself (at least for me).

Thanks!

**Comment by pashashocky** [comment]
@trevware completely agree, this is now available in `0.0.55`, the code-block `showTitle` setting will override what you have set in the Appearance `show inline title` toggle.

**Comment by pashashocky** [comment]
re-open if any issues

**Comment by trevware** [comment]
Looks like this is still not functioning as intended. Using the following code block:

```
~~~~note-gallery
showTitle: true
query: 'tag:#snippet' 
debugQuery: false
recursive: true
limit: 10
sort: desc
sortBy: mtime
fontSize: 12pt
~~~~
``` 

I tried repositioning the arguments, reinstalling via BRAT, but the issue persists. If I toggle `show inline title` on again, the titles will appear with no issue. I don't think there's anything else impacting the plugin. It's not a theme issue either, I've tested it with `Default`. Maybe I'm missing something?
